### PR TITLE
Profiles

### PR DESCRIPTION
**Pull request by jklockri**
Allows user to update email or name from profile section.

### CODE REVIEW

**Comment by zachgarwood** [comment]
Minor changes, but LGTM

File: src/components/profile.vue [hunk 1]
```diff
@@ -18,53 +17,77 @@
...
-                <h3 id="password" class="userInfo input-group mb-3">Password: <span class="gray light">{{ sessionInfo.password }}</span></h3>
-                <div class="input-group input-group-sm mb-3 inputs" v-show="edit === 'on'">
+
+                <h3 id="password" class="userInfo input-group mb-3">Password:</h3>
+                <div class="input-group input-group-sm mb-3 inputs" >
                   <div class="input-group-prepend input-caps">
                     <span class="input-group-text">New</span>
                   </div>
-                  <input id="form_password" type="text" class="form-control" aria-label="Small" aria-describedby="inputGroup-sizing-sm">
+                  <input id="form_password" type="password" class="form-control" aria-label="Small" aria-describedby="inputGroup-sizing-sm">
```
Comment (mkirlin) [inline src/components/profile.vue:65]: What was the reason behind this originally being a text field, and why are we changing it to a password field?

File: src/components/profile.vue [hunk 2]
```diff
@@ -18,53 +17,77 @@
...
                 </div>
-                <div class="input-group input-group-sm mb-3 inputs confirm-inputs" v-show="edit === 'on'">
+
+                <div class="input-group input-group-sm mb-3 inputs confirm-inputs" >
                   <div class="input-group-prepend input-caps">
                     <span class="input-group-text">Confirm</span>
                   </div>
-                  <input id="form_con_password" type="text" class="form-control" aria-label="Small" aria-describedby="inputGroup-sizing-sm">
+                  <input id="form_con_password" type="password" class="form-control" aria-label="Small" aria-describedby="inputGroup-sizing-sm">
```
Comment (mkirlin) [inline src/components/profile.vue:72]: Same ? from ln. 65.

File: src/components/profile.vue [hunk 3]
```diff
@@ -127,30 +151,39 @@
...
+            .then(session => {
+              axios.post(`/api/v1/profile/edit/${this.sessionInfo._id.$oid}`, {
+                headers: { 'x-token': session },
+                params: {
+                  full_name: evt.target.form_name.value.length !== 0 ? evt.target.form_name.value : this.sessionInfo.full_name,
+                  email: evt.target.form_email.value.length !== 0 ? evt.target.form_email.value : this.sessionInfo.email ,
+                }
+            })
+            .then(res => {
+              console.log('DATA RETURNING', res.data)
```
Comment (mkirlin) [inline src/components/profile.vue]: Can we remove this console.log() call? Is there any reason for it to still be in here.

**Pull request merged**